operator/src/main/resources/scripts/startServer.sh

```diff
@@ -286,6 +286,7 @@ if [[ "${KUBERNETES_PLATFORM^^}" == "OPENSHIFT" ]]; then
     fi
 
 fi
+JAVA_OPTIONS="-XX:+CrashOnOutOfMemoryError $JAVA_OPTIONS"
```

Comment:
I assume this may need an "export"?

Reply:
Also, shall we provide a way to remove this "-XX" just in case it has an unancticipated problem? As in something like:

```
JAVA_OPTIONS="${JAVA_XX_OPTION:--XX:+CrashOnOutOfMemoryError} $JAVA_OPTIONS"
```

Reply:
Good point, changed. Thanks.